mayur/Cart/main.rb

```diff
@@ -0,0 +1,57 @@
+load 'inventory.rb'
+load 'shop.rb'
+
+def main
+  inventory = Inventory.new(Array.new,Array.new)
+  loop do
+    print 'Inventory or Shop or End: '
+    track = gets.chomp.downcase
```

Comment:
you can make the variable as choice as it is more readable

Reply:
Thanks for suggestion changes made